preprocessing/tabular_data/preprocessing_tabular_data.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>Suggestion to rephrase the second sentence:</p><p> </p><blockquote>These visualizations helps us analyze correlated features and are particularly important if we want to remove redundant features.</blockquote><p> </p><p>Suggestion to rephrase the last sentence:</p><p> </p><blockquote>The clustergrid is similar, however it presents feature correlations hierarchically.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>